File: demo2_image/demo2_image/ViewController.m
//
//  ViewController.m
//  demo2_image
//
//  Created by ChenXin on 16/9/17.
//  Copyright © 2016年 ChenXin. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UIImageView *topImageView;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    UIImage *image = [UIImage animatedImageNamed:@"animateImage" duration:2.0];
    self.topImageView.image = image;
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: demo1_greating/demo1_greating/ViewController.swift
//
//  ViewController.swift
//  demo1_greating
//
//  Created by ChenXin on 16/9/16.
//  Copyright © 2016年 ChenXin. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var nameField: UITextField!
    
    @IBOutlet weak var messageLabel: UILabel!
    
    @IBAction func greeting(_ sender: AnyObject) {
        self.messageLabel.text = "你好呀，" + self.nameField.text! + "~很高兴认识你~"
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        // 设置App的launch加载时间为1秒
        Thread.sleep(forTimeInterval: 1.0)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

